### ISSUE TITLE
Increase contrast between selected tab and other tabs 

### ISSUE THREAD

**Issue by black7375** [black7375/Firefox-UI-Fix#178]
Copy from https://github.com/black7375/BlaCk-Void-Zsh/issues/28#issue-972976542

---

Thanks a lot for your beautiful theme! It makes the toolbars so much more compact!

Is there any hack to make the currently selected tab lighter so it stands out more between the other tabs? I assume I need to edit `userChrome.css` to change the colour, but could you give me some pointers for the lines to look at?

<img width="745" alt="Screenshot 2021-08-17 at 18 32 05" src="https://user-images.githubusercontent.com/43832342/129784060-b16db560-976b-4d89-bb2a-d3891731b4db.png">

**Comment by black7375** [comment]
You seem to be using light theme.

https://github.com/black7375/Firefox-UI-Fix/blob/319c39dbd07da2ed763a44928803b3f66ffe018c/userChrome.css#L4-L6

---

Example:
![image](https://user-images.githubusercontent.com/25581533/129853419-8716b41e-214d-475f-b9fc-f33b2f6ced8a.png)
```css
  :root[lwtheme-mozlightdark]:not([lwthemetextcolor="bright"]) {
    --lwt-accent-color: rgb(229, 229, 235) !important; /* Original: rgb(240, 240, 244) */
    --toolbar-bgcolor: red !important; /* Use any color you like. */
  }
```

**Comment by samcarter** [comment]
@black7375  Thank you so much! That totally solved it! 

(sorry for posting at the wrong place)
